src/System.Diagnostics.Process/tests/ProcessTests.netstandard.cs

```diff
@@ -66,7 +66,7 @@ public void TestRespondingWindows()
             }
         }
 
-        [PlatformSpecific(TestPlatforms.AnyUnix)]
+        [PlatformSpecific(TestPlatforms.AnyUnix)]  // Uses P/Invokes to get the process Id
```

Comment:
I added this comment since the test reaches [this](https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Unix/System.Native/Interop.GetPid.cs#L13) at some point. Am I missing something? @stephentoub